toolkit.lisp

```diff
@@ -13,6 +13,17 @@
 (defun unix->universal (unix)
   (+ unix *unix-epoch-difference*))
 
+(defun universal->timestring (universal)
```

Comment:
I tried to make this function returns a string for time in UTC. But i am not a date/time expert! :)